docs/config.md

```diff
@@ -247,26 +247,16 @@ heap does not support any options.
 <a name="protocol-http"></a>
 ### HTTP/1.1 protocol parameters
 
-HTTP requests are routed by a combination of Host header, method, and URI.
-Specifically, HTTP/1.0 logical names are of the form:
-```
-  dstPrefix / "1.0" / method [/ uri* ]
-```
-and HTTP/1.1 logical names are of the form:
-```
-  dstPrefix / "1.1" / method / host [/ uri* ]
-```
-
-The default _dstPrefix_ is `/http`. In both cases, `uri` is only considered a part
-of the logical name if the config option `httpUriInDst` is true.
-
 Router configuration options include:
 * *httpAccessLog* -- Sets the access log path.  If not specified, no
 access log is written.
 * *httpUriInDst* -- If `true` http paths are appended to destinations. This allows
```

Comment:
I waffled on this a bit, as I assume most every http identifier will have this option. Ultimately I think you're right, it's specific enough to identifiers that I think it makes sense